num/dualquat/dual_simple_example_test.go

```diff
@@ -0,0 +1,95 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package dualquat_test
+
+import (
+	"fmt"
+
+	"gonum.org/v1/gonum/num/dualquat"
+	"gonum.org/v1/gonum/num/quat"
+)
+
+// Example point, displacement and rotation from Euclidean Space Dual Quaternions page:
```

Comment:
It seems that this is worth including in the example text, the website seems nice.

Reply:
Added in the package doc.